### DISCUSSION TITLE
Handling Multiple Options Selection With OnChange EventListener

### DISCUSSION THREAD

**Discussion by R0LL0** [sa-si-dev/virtual-select#188] [Q&A] [answered]
I am trying to log my VirtualSelect multiple selected options with the onchange event listener in javascript,, it is working if no other CSS and JS files are imported in the HTML file but if i import other styles and js files the onchange event stops working totally. Do you have any suggestions? Or did you face any similar issues?

**Comment by sa-si-dev** [comment, marked as answer]
Please share the code you're using to initiate the plugin and onchange listener.

**Reply by R0LL0** [reply]
Hello, it already worked, the problem is that I was initializing from native select element which is not recommended so I switched my `select` tag to a `div` tag and all worked fine.
Thanks!
